src/Illuminate/Console/Application.php

```diff
@@ -54,11 +55,11 @@ class Application extends SymfonyApplication implements ApplicationContract
     protected static $bootstrappers = [];
 
     /**
-     * A map of command names to classes.
+     * The lazy command loader.
      *
-     * @var array
+     * @var \Illuminate\Console\ContainerCommandLoader
```

Comment:
```suggestion
     * @var \Symfony\Component\Console\CommandLoader\CommandLoaderInterface
```

Since you are setting this attribute in `setContainerCommandLoader(CommandLoaderInterface $loader = null)`

It should also be of the same Interface.

Reply:
Thanks for the suggestion!

Reply:
I've committed your suggestion - is this pretty common to do across the framework to use interfaces in docblocks rather than reference actual Illuminate components?

Reply:
I am not sure about this one.
I just saw that you are setting an Interface on a variable which says it should be an implementation of that interface.
And in my personal Opinion this should not happen. Otherwise the Docblock or the arguments on `setContainerCommandLoader` should also be the implementation (would be meaningless otherwise)